guides/common/modules/proc_using-upstream-server-as-a-content-store.adoc

```diff
@@ -31,3 +31,12 @@ For more information on performing an incremental Library export, see xref:Expor
 . Import the content to an organization using the procedure outlined in xref:Importing_into_the_Library_Environment_{context}[].
 +
 You can then manage content using Content Views or Lifecycle Environments as you require.
+
++
+[NOTE]
+====
+When you are unable to match upstream and downstream {Project} versions, you can use:
```

Comment:
There's no context here explaining _that_ you need to match the versions, or why. I feel like the place for this note is alongside that missing context, probably above this section where it describes import/export more generally.